CMakeLists.txt

```diff
@@ -110,7 +110,7 @@ add_message_files(
 
 ## Generate dynamic reconfigure parameters in the 'cfg' folder
 generate_dynamic_reconfigure_options(
-        cfg/DetectorParams.cfg
+        config/DetectorParams.cfg
```

Comment:
do we need to add in the other config files here or no?

Reply:
No, I think we only need this one here bc it's used to reconfigure the AR tag detector at runtime even tho its cpp. The other config files don't need to be in the cmake file, right @qhdwight?

Reply:
Correct